x/slashing/abci.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/slashing/types"
 )
 
+var deprecatedBitArrayPruneLimitPerBlock = 2000
```

Comment:
I could be convinced to lower this, but I would rather have the one time payment of deleting every block frontloaded at time of upgrade rather than slight degradation for a very long time.

Reply:
I would agree with this